cypress/utils/index.js

```diff
@@ -27,5 +27,5 @@ export const hasTatkalAlreadyOpened = (TRAIN_COACH) => {
 
 export function tatkalOpenTimeForToday(TRAIN_COACH) {
     const openTime = tatkalOpenTimings[TRAIN_COACH]
-    return `${dayjs().format('DD-MMM-YYYY')} [${openTime}]`
+    return `[${openTime}`
```

Comment:
@drtechie  can you fix this typo here square brackets

Reply:
i am fixing this...

Reply:
We are checking for [10:00:X]. X can 01, 02 etc. 
So the exact text that we will check is [10:00.

The closing square bracket I omitted intentionally. Travelling now. Will run and check again once.

Reply:
Basically as soon as the time changes to 10:00:<any second>, we make the click.

Reply:
but this behavior would restrict the code to work only for tatkal right beside that any time this whole bunch won't work just in case letting you know.... 
![image](https://github.com/shivamguys/irctc-cypress-automation/assets/25263989/ac180bf0-0c90-4b17-8c29-ec6e545f3038)

Reply:
Missed removing :ss from date format. 
Please check #17
Will rerun the scenarios once again.